applications/salk-portal/package-lock.json

```diff
@@ -1812,6 +1812,24 @@
         "@turf/helpers": "^6.5.0"
       }
     },
```

Comment:
@paularah please remove this file, do not push this, keep only on your local env